src/apm/storage/ipfs.js

```diff
@@ -0,0 +1,29 @@
+const ipfsAPI = require('ipfs-api')
+const streamToString = require('stream-to-string')
+
+const ipfs = ipfsAPI()
```

Comment:
Does this try to connect to a local ipfs instance by default? We should have a flag for configuring it and allow to set it to public ipfs nodes.

Reply:
Yes, it does. Generally, we have an issue in passing configuration to APM storage providers.

Do you think something like `--<provider-name>.<option>=<value>` would work (which could also be set in the configuration from #25) or do you prefer something like `--<provider-name>-config=<json object>` or  `--<provider-name>-config=<path to json file>`?

Reply:
Just saw this, sorry. I think `--<provider-name>.<option>=<value>` and allowing it to be set in the config files is a good idea